chapter_25_CI.asciidoc

```diff
@@ -984,6 +1010,21 @@ And if you navigate through, you'll see something like <<gitlab_ui_show_screensh
 [[gitlab_ui_show_screenshot]]
 image::images/gitlab_ui_show_screenshot.png["GitLab UI showing a normal-looking screenshot of the site"]
 
+// CSANAD: working from the book-example, this is not true for me:
+// https://gitlab.com/csberes/superlistz-version-harry/-/jobs/9765894394
+// ```
+// WARNING: src/functional_tests/screendumps/: no matching files. Ensure that the artifact path is relative to the working directory (/builds/csberes/superlistz-version-harry)
+// ERROR: No files to upload
+// Cleaning up project directory and file based variables 00:00
+// Job succeeded
+// ```
+// The path looks OK, and locally, the test does create a `screenshots` directory.
+// The directory does not get created in the Gitlab build job - I guess,
+// because there are no errors. So, even though in my VM the official book-example
+// version does generate a screenshot for every FT, it isn't true for the Gitlab.
+// However, there is another odd thing: I tried re-adding the `self.fail`
+// which resulted in three screenshots on Gitlab:
+// https://gitlab.com/csberes/superlistz-version-harry/-/jobs/9766298077
```

Comment:
Three screenshots, but only one failing test, which I made to fail.
![Screenshot from 2025-04-19 00-33-56](https://github.com/user-attachments/assets/a415e5a9-880b-48ec-9b11-d890e8747a2f)